contracts/splits/src/contract.rs

```diff
@@ -35,7 +35,10 @@ pub fn instantiate(
 
     set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
 
-    let cfg = Config { group_addr };
+    let cfg = Config {
+        group_addr,
+        executor: msg.executor,
```

Comment:
address is not being validated if it is Only(Addr)

Reply:
we should set sane defaults, and i think the default should be member-gated distribute instead of anyone can distribute.